tradeit-android-sdk/src/main/kotlin/it/trade/android/sdk/model/TradeItCryptoTradeOrderDetailsParcelable.kt

```diff
@@ -0,0 +1,65 @@
+package it.trade.android.sdk.model
+
+import android.os.Parcel
+import android.os.Parcelable
+import it.trade.model.reponse.CryptoTradeOrderDetails
+
+class TradeItCryptoTradeOrderDetailsParcelable : Parcelable {
```

Comment:
could you please also add `toString` method to this parcelable? like the others.